models/media_ad_views/scratch/snowplow_media_player_media_ad_views_this_run.sql

```diff
@@ -44,17 +44,39 @@ events_this_run as (
     ,ev.media_ad_break__break_id as ad_break_id
     ,ev.media_ad__ad_id as ad_id
 
+
+    {%- set non_agg_columns = [] -%}
+    {%- for identifier in var('snowplow__ad_views_passthroughs', []) -%}
+      {%- if identifier is not mapping or 'agg' not in identifier -%}
+        {%- do non_agg_columns.append(identifier) -%}
+      {%- endif -%}
+    {%- endfor -%}
+
+
     {%- if var('snowplow__ad_views_passthroughs', []) -%}
       {%- set passthrough_names = [] -%}
+      {%- set agg_columns = [] -%}
```

Comment:
It might just be me but calling it agg_columns is confusing, can it be agg_identifiers?